discord/discordDeleteMessage.html

```diff
@@ -0,0 +1,55 @@
+<script type="text/javascript">
+  RED.nodes.registerType('discordDeleteMessage', {
+    category: 'discord',
+    color: '#7289da',
+    defaults: {
+      name: {
+        value: "",
+        required: false
+      },
+      token: {
+        value: "",
+        required: true,
+        type: "discord-token"
+      }
+    },
```

Comment:
The `discordSendMessage` provides a mechanism for setting the channel id within the node itself as a configuration parameter, whilst `discordDeleteMessage` does not. Either works for me, though the lack of consistency here is not ideal, and something that I think would be good to address.

The way I see it, we have two options here:
1. Remove the ability in `discordSendMessage` to specify the channel in the node config. This would be a breaking change that needs to be mentioned in the changelog.
2. Add the ability in `discordDeleteMessage` to specify the channel id, in the same way as the `discordSendMessage` node allows it.

Personally I'd lean to option 2 to avoid breaking changes, but again, I'm open to either, especially considering this changeset already brings breaking changes with it.